### PR TITLE
promscale: 0.6.2 -> 0.7.1

### PR DESCRIPTION
**Pull request by JamesGuthrie**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by iAmMrinal0** [comment]
Result of `nixpkgs-review pr 149609` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>promscale (timescale-prometheus)</li>
  </ul>
</details>

File: pkgs/servers/monitoring/prometheus/promscale/default.nix [hunk 1]
```diff
@@ -29,6 +29,10 @@ buildGoModule rec {
     fi
   '';
```
Comment (0x4A6F) [inline pkgs/servers/monitoring/prometheus/promscale/default.nix:30]: Would be nice to also remove `doInstallCheck` and `installCheckPhase` and move to `testVersion` in this update:
```diff
 { lib
 , buildGoModule
 , fetchFromGitHub
+, promscale
+, testVersion
 }:

-  doInstallCheck = true;
-  installCheckPhase = ''
-    if [[ "$("$out/bin/${pname}" -version)" == "${version}" ]]; then
-      echo '${pname} smoke check passed'
-    else
-      echo '${pname} smoke check failed'
-      exit 1
-    fi
-  '';

+  passthru.tests.version = {
+    package = promscale;
+    command = "promscale -version";
+  };

```

File: pkgs/servers/monitoring/prometheus/promscale/default.nix [hunk 2]
```diff
@@ -29,6 +29,10 @@ buildGoModule rec {
...
 
+  patches = [
+    ./0001-remove-jaeger-test-dep.patch
+  ];
+
```
Comment (0x4A6F) [inline pkgs/servers/monitoring/prometheus/promscale/default.nix:35]: Suggestion:
patches reside directly under src.

**Review by 0x4A6F — APPROVED** [review]
Thanks for your fixes!

You may incorporate my suggestions, otherwise LGTM.

File: pkgs/servers/monitoring/prometheus/promscale/default.nix (hunk 2 above)
Comment (JamesGuthrie) [reply pkgs/servers/monitoring/prometheus/promscale/default.nix:35]: I'm sorry, I don't understand this suggestion.

File: pkgs/servers/monitoring/prometheus/promscale/default.nix (hunk 1 above)
Comment (JamesGuthrie) [reply pkgs/servers/monitoring/prometheus/promscale/default.nix:30]: 👍

File: pkgs/servers/monitoring/prometheus/promscale/default.nix (hunk 2 above)
Comment (JamesGuthrie) [reply pkgs/servers/monitoring/prometheus/promscale/default.nix:33]: Ah, I think that I figure this out.

Comment (0x4A6F) [reply pkgs/servers/monitoring/prometheus/promscale/default.nix:33]:
```
--- a/pkgs/servers/monitoring/prometheus/promscale/default.nix
+++ b/pkgs/servers/monitoring/prometheus/promscale/default.nix
@@ -14,6 +14,10 @@ buildGoModule rec {
     sha256 = "sha256-OMDl8RGFOMW+KNX2tNHusJY/6gLZxuWCI3c0E/oqrfE=";
   };

+  patches = [
+    ./0001-remove-jaeger-test-dep.patch
+  ];
+
   vendorSha256 = "sha256-IwHngKiQ+TangEj5PcdiGoLxQJrt/Y3EtbSYZYmfUOE=";

   ldflags = [ "-s" "-w" "-X github.com/timescale/promscale/pkg/version.Version=${version}" "-X github.com/timescale/promscale/pkg/version.CommitHash=${src.rev}" ];
@@ -29,10 +33,6 @@ buildGoModule rec {
     fi
   '';

-  patches = [
-    ./0001-remove-jaeger-test-dep.patch
-  ];
-
   meta = with lib; {
     description = "An open-source analytical platform for Prometheus metrics";
     homepage = "https://github.com/timescale/promscale";
```

**Pull request merged**